app/src/org/commcare/android/javarosa/IntentCallout.java

```diff
@@ -316,7 +317,10 @@ private void storePointerToFileResponse(TreeReference ref, String responseValue,
 
             //Wipe out any reference that exists
             formDef.setValue(null, ref);
-        } catch (IOException ioe) {
+        } catch (FileExtensionNotFoundException e) {
+            Logger.exception(LogTypes.TYPE_ERROR_STORAGE, e);
+            formDef.setValue(null, ref);
+        }  catch (IOException ioe) {
             ioe.printStackTrace();
             Log.e(TAG,"CommCare failed to copy a binary input from an intent callout from: " + responseValue);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/41ae22492f9a1ba7ab597c2d29700051b5a1c31a/app/src/org/commcare/android/javarosa/IntentCallout.java#L325"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>